src/main/java/mate/academy/biblio/model/Book.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.biblio.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "book")

```

Comment:
according to the naming convention table name should be plural
```suggestion
@Table(name = "books")
```